src/mongo/tree/MongoCollectionTreeItem.ts

```diff
@@ -35,13 +35,13 @@ export class MongoCollectionTreeItem implements IAzureParentTreeItem {
 		}
 	}
 
-	public async update(documents: IMongoDocument[]): Promise<IMongoDocument[]> {
+	public async update(documents: IMongoDocument[], flag = false): Promise<IMongoDocument[]> {
```

Comment:
Why the changes to this file? I don't see this being called anywhere in this PR

Reply:
This must have come from an earlier commit which was later undone. I'll fix this.